doc/source/dev/missing-bits.rst

```diff
@@ -48,6 +48,46 @@ additional parameters can be added to the function anywhere after the
 ``*``; new parameters do not have to be added after the existing parameters.
 
 
+Return Objects
+~~~~~~~~~~~~~~
+For new functions or methods that return two or more conceptually distinct
+elements, return the elements in an object type that is not iterable. In
+particular, do not return a ``tuple``, ``namedtuple``, or a "bunch" produced
+by `~scipy._lib._bunch.make_tuple_bunch`, the latter being reserved for adding
```

Comment:
```suggestion
by ``scipy._lib._bunch.make_tuple_bunch``, the latter being reserved for adding
```
This one doesn't has documentation, so it renders in italics. I think that monospace font (like code) is preferable.